src/backgroundScripts/model.js

```diff
@@ -87,10 +98,33 @@ const model = (function () {
 		return false
 	}
 
+  const renameTopic = (vpId,topicId,newName) => {
+    return db.get({_id:vpId})
```

Comment:
`db` is not defined which prevents renaming feature to work. 
This feature should be adapted to work with the user defined Hypertopic service.

Reply:
In case it can help, I was also working on this merge on this branch https://github.com/franck-eyraud/LaSuli/tree/feature-settings 

You can find [here](https://github.com/franck-eyraud/LaSuli/commit/90f7d3cf4f5fd87cdaff978176cec5386d53ceb3#diff-2b6e46436cc8167726516dc3bacdf6acR113) how the db is taken from the settings.

Reply:
Thank you @franck-eyraud :) But the message was to Triinu so that she can complete her assignment ^^